webpack/assets/javascripts/react_app/components/HostStatus/HostStatus.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { OverlayTrigger, Popover } from 'patternfly-react';
+import classNames from 'classnames';
+import './HostStatus.scss';
+
+const HostStatus = ({ className, overviewFields }) => {
+  const cx = classNames(className, 'clickable-host-status');
+  const table = (
+    <table id="properties_table">
+      <tbody>
+        {overviewFields.map(([name, value], key) => (
+          <tr key={key}>
+            <td className="property_name">{name}</td>
+            <td
+              className="property_value"
+              dangerouslySetInnerHTML={{ __html: value }}
```

Comment:
instead of dangerously render html content, the data can be loaded with an API call or a dedicated lean controller endpoint (but only when the mouse hovers more than few seconds for optimization reason or a click triggering)

Reply:
sounds right to me, I also commented about that earlier over [here](https://github.com/theforeman/foreman/pull/7881#issuecomment-669181073)
This is just the draft stage to get more feedback